contracts/core/fee-calculators/FixedFeeCalculator.sol

```diff
@@ -79,6 +79,16 @@ contract FixedFeeCalculator is IFeeCalculator {
         return fees[msg.sender];
     }
 
+    /**
```

Comment:
Do we want to version this file `FixedFeeCalculatorV2.sol` so that it's clear this is an updated version from the original one we deployed?

Reply:
Currently leaning towards leaving it - as deployed versions are not currently utilized and we plan not to going forward